docs/dataset/formatting-your-datasets.md

```diff
@@ -90,13 +98,51 @@ These annotations are visible on both the Gallery view for groups of datapoints
 
 ### Structured Data
 
+Consider a `.csv` file that contains ground truth data (labeled bounding boxes) for an object detection problem:
+
+| locator                                                                       | label      | min_x     | max_x  | min_y | max_y   |
+|-------------------------------------------------------------------------------|------------|-----------|--------|-------|---------|
+| s3://kolena-public-examples/coco-2014-val/data/COCO_val2014_000000369763.jpg | motorcycle | 270.77    | 621.61 | 44.59 |  254.18  |
+| s3://kolena-public-examples/coco-2014-val/data/COCO_val2014_000000369763.jpg | car        | 538.03    | 636.85 | 8.86  | 101.93  |
+| s3://kolena-public-examples/coco-2014-val/data/COCO_val2014_000000369763.jpg | trunk      | 313.02    | 553.98 | 12.01 | 99.84   |
+
+The bounding box for the first image is `(270.77, 44.59), (621.61,  254.18)`. To represent this within Kolena use the
+[`BoundingBox`](../reference/annotation.md#kolena.annotation.BoundingBox) annotation. This looks like:
+
+```python
+from kolena.annotation import BoundingBox
+bbox = BoundingBox(top_left=(270.77, 44.59), bottom_right=(621.61,  254.18))
+```
+When viewing a bounding box within python the format is
+`BoundingBox(top_left=(270.77, 44.59), bottom_right=(621.61, 254.18), width=350.84, height=209.59, area=73532.5556, aspect_ratio=1.67)`
+
+The above example has multiple objects within a single image, in order to ingest this you would need a single field
+containing the list of bounding boxes.
+
+For example:
+```python
+from kolena.annotation import BoundingBox
+bboxes = [
+    BoundingBox(top_left=(270.77, 44.59), bottom_right=(621.61, 254.18)),
+    BoundingBox(top_left=(538.03, 8.86), bottom_right=(636.85, 101.93)),
+    BoundingBox(top_left=(313.02, 12.01), bottom_right=(553.98, 99.84)),
+]
+```
+This would be represented within a csv as show below. Note this will be a single line,
+but is shown here as multiple lines for formatting.
+```
```

Comment:
Thinking about the best way to show this is a single row. Needed to split for both the markdown length rules and the fact that a single line this long does not have a good experience for the reader.

Reply:
What's the benefit in dumping this blob in the documentation? Can't we just say the Kolena annotation class is serialized to json or something

Reply:
The idea was to show the user how to go from the raw data to a .csv that can be uploaded. The purpose of adding the blob was to help users understand how the json itself looks, and to have an idea of how the data looks at each step.